src/PayPal/Provider.php

```diff
@@ -67,7 +67,8 @@ protected function mapUserToObject(array $user)
     {
         return (new User())->setRaw($user)->map([
             'id'       => str_replace('https://www.paypal.com/webapps/auth/identity/user/', null, $user['user_id']),
```

Comment:
Know this would be a breaking change, but why is the ID a url, and not just the user's id?

Reply:
Good question, wish I knew, but that's just how it's returned from the API: https://developer.paypal.com/docs/api/identity/v1/#userinfo_get

In my implementation I actually return the "payer_id" as the ID (which requires the "https://uri.paypal.com/services/paypalattributes" scope) and is much more useful, at least for my purposes, but I didn't want to make a breaking change.